DEPENDENCIES

```diff
@@ -169,7 +169,7 @@ npm/npmjs/-/cssom/0.3.8, MIT, approved, clearlydefined
 npm/npmjs/-/cssom/0.4.4, MIT, approved, clearlydefined
 npm/npmjs/-/cssstyle/2.3.0, MIT, approved, clearlydefined
 npm/npmjs/-/csstype/3.1.1, MIT, approved, clearlydefined
-npm/npmjs/-/cx-portal-shared-components/1.4.0, Apache-2.0 AND CC-BY-4.0 AND OFL-1.1, approved, #9102
+npm/npmjs/-/cx-portal-shared-components/1.5.0, , restricted, clearlydefined
```

Comment:
You have now a restricted component, we need to open an IP issue ticket for that.

Reply:
@adityagajbhiye9  Created an IP issue ticket [here](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/10235). Please keep track of it's status and update your dependencies file when it's approved.

Reply:
we updated dependency file.